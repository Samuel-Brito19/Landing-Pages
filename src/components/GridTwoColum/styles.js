import styled, { css } from 'styled-components'

export const Container = styled.div`
${({ theme }) => css`
display: grid;
grid-template-columns: 1fr 2fr;
align-items: center;
gap: ${theme.spacings.large};
`}
`;

export const TextContainer = styled.div`
${({ theme }) => css``}
`;

export const ImageContainer = styled.div`
${({ theme }) => css``}
`;

export const Image = styled.img`
${({ theme }) => css``}
`;
