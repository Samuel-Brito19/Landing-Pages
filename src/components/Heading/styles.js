import { css, styled } from "styled-components";

export const Title = styled.h1`
${({ theme }) => css`
color: ${theme.colors.mainBg};

`};

`
