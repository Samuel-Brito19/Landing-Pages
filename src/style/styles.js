import styled, { css } from 'styled-components';


export const Wrap = styled.div`
  ${({ theme }) => css`
    background: ${theme.colors.mainBg};
  `}
`;
