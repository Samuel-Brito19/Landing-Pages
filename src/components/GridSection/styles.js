import styled, { css } from 'styled-components'
import { Title as HeadingContainer } from '../Heading/styles'

export const Container = styled.div`
${({ theme }) => css``}
`;

export const Grid = styled.div`
${({ theme }) => css`
counter-reset: grid-counter;
display: grid;
grid-template-columns: repeat(auto-fill, minMax(280px, 1fr));
gap: ${theme.spacings.large};
`}
`;

export const GridElement = styled.div`
${({ theme }) => css`
${HeadingContainer} {
  position: relative;
  left: 5rem;

}

${HeadingContainer}::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: ${theme.font.sizes.xhuge};
  top: -2rem;
  left: -5rem;
  transform: rotate(5deg);


}
`}
`;
