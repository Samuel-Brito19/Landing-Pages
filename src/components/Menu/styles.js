import styled, { css } from 'styled-components'
import { Container as SectionContainer } from '../SectionContainer/styles';
import { Title as Heading } from '../Heading/styles';

export const Container = styled.div`
${({ theme }) => css`

position: fixed;
z-index: 5;
top: 0;
bottom: 0;
right: 0;
left: 0;
border-bottom: ${theme.colors.mediumGray};
background: ${theme.colors.white};

> ${SectionContainer} {
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: 1fr;
  height: 100vh;
  align-items: center;
  overflow-y: auto;
}

& ${Heading} {
  padding-bottom: ${theme.spacings.large};
  display: flex;
  justify-content: center;
}

@media ${theme.media.lteMedium} {
  height: 100vh;
}

`}
`;

export const MenuContainer = styled.div`
${({ theme }) => css`
display: flex;
justify-content: space-between;
align-items: center;

@media ${theme.media.lteMedium} {
  display: block;
  text-align: center;
  padding: ${theme.spacings.xxlarge} 0;
}

`}
`

export const Button = styled.div`
${({ theme }) => css`
z-index: 6;
position: fixed;
top: 2rem;
right: 2rem;
width: 4rem;
height: 4rem;
background: ${theme.colors.primaryColor};
color: ${theme.colors.white};
border: none;

`}

@media ${theme.media.lteMedium} {
  display: flex;
  align-items: center;
  justify-content: center;
}


> svg {
  width: 2.5rem;
  height: 2.5rem
}
`
