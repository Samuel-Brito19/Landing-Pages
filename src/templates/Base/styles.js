import styled, { css } from 'styled-components'
import { Container as TextComponent } from '../../components/TextComponent/styles';

export const Container = styled.div`
${({ theme }) => css`
& ${TextComponent} {
  font-size: ${theme.font.sizes.small};
}


`}
`;
