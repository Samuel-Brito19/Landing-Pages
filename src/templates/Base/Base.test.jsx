import { screen } from '@testing-library/react'
import { Base } from '.'
import { renderTheme } from '../../templates/styles/renderTheme'
describe('<Base/>', () => {
  it('should render', () => {
    renderTheme(<Base>children</Base>)
    expect(screen.getByRole('heading')).toBeInTheDocument()
  })
})
