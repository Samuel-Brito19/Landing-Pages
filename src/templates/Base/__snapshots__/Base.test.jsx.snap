// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base/> should render 1`] = `
.c11 .c15 {
  font-size: 1.6rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c7 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: none;
}

.c14 {
  color: #0A1128;
  font-size: 6.4rem;
  text-transform: none;
}

.c18 {
  color: #0A1128;
  font-size: 3.2rem;
  text-transform: none;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  background: #FFFFFF;
  transition: all 300ms ease-in-out;
}

.c2 >.c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 .c6 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.c0 {
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  width: 4rem;
  height: 4rem;
  background: #0A1128;
  color: #FFFFFF;
  border: none;
  display: none;
  pointer-events: all;
}

.c0 >svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c8 {
  text-decoration: none;
}

.c8 color:inherit>img {
  max-height: 3rem;
}

.c9 {
  display: flex;
  flex-flow: row wrap;
}

.c10 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c20 {
  text-align: center;
  border-top: 0.1rem solid;
}

.c20 a {
  color: inherit;
  text-decoration: none;
}

.c20 .c15 {
  font-size: 1.6rem;
}

.c20 .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c21 {
  background: #0A1128;
  color: #FFFFFF;
  display: flex;
  align-items: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
  opacity: 0.7;
}

.c13 .c15 {
  margin-bottom: 6.4rem;
}

.c16 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill, minMax(280px, 1fr));
  gap: 3.2rem;
  overflow: hidden;
  width: 100%;
}

.c17 .c6 {
  position: relative;
  left: 5rem;
}

.c17 .c6::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 6.4rem;
  top: -2rem;
  left: -5rem;
  transform: rotate(5deg);
}

.c12 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c19 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

@media (max-width: 768px) {
  .c14 {
    font-size: 4.0rem;
  }
}

@media (max-width: 768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
  }

  .c2 >.c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    align-items: center;
    overflow-y: auto;
  }

  .c2 .c6 {
    padding-bottom: 3.2rem;
    display: flex;
    justify-content: center;
  }
}

@media (max-width: 768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width: 768px) {
  .c0 {
    display: flex;
    align-items: center;
    justify-content: center;
  }
}

@media (max-width: 768px) {
  .c9 {
    flex-flow: column wrap;
    align-content: center;
  }
}

@media (max-width: 768px) {
  .c16 {
    grid-template-columns: 1fr;
  }
}

<div>
  <button
    aria-label="Open/Close menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="Home"
          >
            LOGO
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="#target1"
            target="_self"
          >
            <h1>
              Link 1
            </h1>
          </a>
          <a
            class="c10"
            href="#target2"
            target="_self"
          >
            <h1>
              Link 2
            </h1>
          </a>
          <a
            class="c10"
            href="#target3"
            target="_self"
          >
            <h1>
              Link 3
            </h1>
          </a>
          <a
            class="c10"
            href="#target4"
            target="_self"
          >
            <h1>
              Link 4
            </h1>
          </a>
          <a
            class="c10"
            href="#target5"
            target="_self"
          >
            <h1>
              Link 5
            </h1>
          </a>
          <a
            class="c10"
            href="#target6"
            target="_self"
          >
            <h1>
              Link 6
            </h1>
          </a>
          <a
            class="c10"
            href="#target7"
            target="_self"
          >
            <h1>
              Link 7
            </h1>
          </a>
          <a
            class="c10"
            href="#target8"
            target="_self"
          >
            <h1>
              Link 8
            </h1>
          </a>
          <a
            class="c10"
            href="#target9"
            target="_self"
          >
            <h1>
              Link 9
            </h1>
          </a>
          <a
            class="c10"
            href="#target10"
            target="_self"
          >
            <h1>
              Link 10
            </h1>
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c19"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <div
            class="c15 "
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c15 "
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c20"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c15 "
        >
          <p>
            Teste de footer
          </p>
        </div>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c21"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
